gatekeeperv2config.json

```diff
@@ -0,0 +1,143 @@
+[
+  {
+    "DisplayName": "Discord Bot Token",
+    "Category": "Bot Settings",
+    "Description": "Enter your [Discord Bot Token](https://discord.com/developers/applications/).",
+    "Keywords": "discord,token",
+    "FieldName": "token",
+    "InputType": "password",
+    "IsFlagArgument": false,
+    "ParamFieldName": "token",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "Placeholder": "319/f0dsA0jk0PKge2jkpSFWEauopbvejwaioj32omoi32JIOabdsajotjewq3jr2=="
+  },
+  {
+    "DisplayName": "Discord Server Guild ID",
+    "Category": "Bot Settings",
+    "Description": "Enter your Discord server's [Guild ID](https://poshbot.readthedocs.io/en/latest/guides/backends/setup-discord-backend/#find-your-guild-id-server-id).",
+    "Keywords": "discord,guild,id",
+    "FieldName": "guildID",
+    "InputType": "number",
+    "IsFlagArgument": false,
+    "ParamFieldName": "guildID",
+    "IncludeInCommandLine": false,
+    "DefaultValue": "",
+    "Placeholder": "883220743755206666"
+  },
+  {
+    "DisplayName": "AMP User",
+    "Category": "Bot Settings",
+    "Description": "The Login Name for the account you want the bot to use. This account must have \"Super Admin\" privelege for setup of new instances and after upgrades.",
```

Comment:
Typo in "privilege"

Reply:
I was using the... America Midwest form of the word. I'll adjust it to match the International spelling.